src/Core/Resolvers/CosmosQueryBuilder.cs

```diff
@@ -125,7 +129,11 @@ protected override string Build(Predicate? predicate)
             }
 
             string predicateString;
-            if (ResolveOperand(predicate.Right).Equals(GQLFilterParser.NullStringValue))
+            if (predicate.Op == PredicateOperation.ARRAY_CONTAINS || predicate.Op == PredicateOperation.NOT_ARRAY_CONTAINS)
```

Comment:
Are there potentially more ARRAY operators which might need to be added soon? If so, should we add a helper which classifies an operator as an ARRAY operator rather than appending an `||` for each of them?

Reply:
I don't think there are more. if in future, I need to add more such condition, I will definitely consider it to move to a helper function.